### PR TITLE
update VF VCF writer to remove common trailing bases from SPDI

### PR DESCRIPTION
**Pull request by sarahhunt**
Used to create VCF line on web site

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Utils/Sequence.pm [hunk 1]
```diff
@@ -1001,6 +1002,63 @@ sub trim_sequences {
...
+  Arg[0]      : arrayref of allele sequencess
+  Example     : my $trimmed_alleles = @{trim_right(\@alleles)}
+  Description : Takes a set of allele sequences and trims common sequence
+                from the end.
+                Reduces fully justified allele strings for VCF allele writing
+                Handles multi-alleleic variants
+                Stops when an allele sequence has length 1 (common bases
+                at the start are not removed to support VCF)
+
+  ReturnType  : arrayref of timmed allele sequences
```
Comment (at7) [inline modules/Bio/EnsEMBL/Variation/Utils/Sequence.pm]: typo timmed -> trimmed

**Comment by at7** [comment]
Just a comment, that the code is also used by VEP's OutputFactory.

File: modules/Bio/EnsEMBL/Variation/Utils/Sequence.pm [hunk 2]
```diff
@@ -1001,6 +1002,63 @@ sub trim_sequences {
...
+=cut
+
+sub trim_right{
+
+  my $alleles = shift;
+
+  throw("Allele sequences required") unless scalar(@{$alleles}) > 0;
+
+  ## do nothing if there is only one allele
+  return $alleles if scalar(@{$alleles}) == 1;
```
Comment (helensch) [inline modules/Bio/EnsEMBL/Variation/Utils/Sequence.pm]: Suggest add test for this @alleles = ('A');

**Comment by at7** [comment]
merged with master and release/98

**Pull request merged**